packages/core-commands/src/site-editor-navigation-commands.js

```diff
@@ -172,12 +172,16 @@ const getNavigationCommandLoaderPerTemplate = ( templateType ) =>
 				const extraArgs = isSiteEditor
 					? { canvas: getQueryArg( window.location.href, 'canvas' ) }
 					: {};
+				const templateLabel =
+					templateType === 'wp_template_part'
+						? __( 'template part' )
+						: __( 'template' );
 
 				return {
 					name: templateType + '-' + record.id,
 					searchLabel: record.title?.rendered + ' ' + record.id,
 					label: record.title?.rendered
-						? record.title?.rendered
+						? record.title?.rendered + ' ' + templateLabel
```

Comment:
I think this should probably use `sprintf` as otherwise this is not translatable as it should. It's not a suffix in all languages.

Something like:

```
const templateLabel =
					templateType === 'wp_template_part'
						? __( '%s template part' )
						: __( '%s template' );
						
						label = sprintf( templateLabel, title );
```

Reply:
Added in https://github.com/WordPress/gutenberg/pull/54366/commits/1421b3577b38e5ca245943de1c03635f44f2b110.